percona/controller/pgcluster/controller.go

```diff
@@ -418,3 +447,36 @@ func isBackupRunning(ctx context.Context, cl client.Reader, cr *v2.PerconaPGClus
 
 	return false, nil
 }
+
+func (r *PGClusterReconciler) startExternalWatchers(ctx context.Context, cr *v2.PerconaPGCluster) []string {
+	log := logging.FromContext(ctx)
+
+	watcherName, watcherFunc := watcher.GetWALWatcher(cr)
+	if !r.Watchers.IsExist(watcherName) {
+		log.Info("Starting WAL watcher", "name", watcherName)
+
+		if err := r.Watchers.Add(watcherName, watcherFunc); err != nil {
+			return nil
+		}
+
+		go watcherFunc(ctx, r.Client, r.ExternalChan, r.StopExternalWatchers, r.PodExec, cr)
+	}
+
+	return r.Watchers.Names()
```

Comment:
Return value is not used anywhere

Reply:
https://github.com/percona/percona-postgresql-operator/pull/759/commits/4127c98394e9066443f18fa6d69ce23a46ac10a6